frontend/src/pages/AdminLogin.jsx

```
@@ -20,11 +20,11 @@ const AdminLogin = () => {
     setError('');
 
     try {
-      const response = await fetch('http://localhost:4000/api/auth/admin/login', {
+      const response = await fetch(`${import.meta.env.VITE_API_URL}/api/auth/admin/login`, {
         method: 'POST',
         headers: { 'Content-Type': 'application/json' },
-        body: JSON.stringify(formData),
-        credentials: 'include', // Ensure cookies (if used) are included
+        body: JSON.stringify(data),
```

Biome (1.9.4)

[error] This variable is used before its declaration.

The variable is declared here:


(lint/correctness/noInvalidUseBeforeDeclaration)